tests/test_voc_format.py

```diff
@@ -496,6 +537,228 @@ def test_can_pickle(self):
                 compare_datasets_strict(self, source, parsed)
 
 
+class VocExtractorTest(TestCase):
+    # ?xml... must be in the file beginning
+    XML_ANNOTATION_TEMPLATE = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
```suggestion
    XML_ANNOTATION_TEMPLATE = """\
<?xml version="1.0" encoding="UTF-8"?>
```

Just for the sake of readability. You could also use `textwrap.dedent` to avoid violating natural indentation for the contents of the string.

Reply:
Agree with \\. 
Indentation is intended for simplified editing of this fragment.